tensorboard/plugins/custom_scalar/custom_scalars_plugin.py

```diff
@@ -276,7 +276,10 @@ def layout_impl(self):
           if category.title in title_to_category:
             # A category with this name has been seen before. Do not create a
             # new one. Merge their charts.
```

Comment:
Could you extend the comment here to say "Merge their charts, skipping any duplicates."?